spec/features/items/index_spec.rb

```diff
@@ -58,5 +58,120 @@
       end
         expect(current_path).to eq("/items/#{@pull_toy.id}")
     end
+
+    describe "When I visit the items index page to see item stats" do
+      before(:each) do
+        @paper = @meg.items.create(name: "Lined Paper", description: "Great for writing on!", price: 20, image: "https://cdn.vertex42.com/WordTemplates/images/printable-lined-paper-wide-ruled.png", inventory: 3)
+        @pencil = @meg.items.create(name: "Yellow Pencil", description: "You can write on paper with it!", price: 2, image: "https://images-na.ssl-images-amazon.com/images/I/31BlVr01izL._SX425_.jpg", inventory: 100)
+        @candy = @meg.items.create(name: "Yellow candy", description: "You can eat it!", price: 3, image: "https://images-na.ssl-images-amazon.com/images/I/610ZOacuGPL._SL1050_.jpg", inventory: 100)
+        @okay_paper = @meg.items.create(name: "Okay Paper", description: "Great for writing on!", price: 20, image: "https://cdn.vertex42.com/WordTemplates/images/printable-lined-paper-wide-ruled.png", inventory: 3)
+        @okay_pencil = @meg.items.create(name: "Okay Pencil", description: "You can write on paper with it!", price: 2, image: "https://images-na.ssl-images-amazon.com/images/I/31BlVr01izL._SX425_.jpg", inventory: 100)
+        @okay_candy = @meg.items.create(name: "Okay candy", description: "You can eat it!", price: 3, image: "https://images-na.ssl-images-amazon.com/images/I/610ZOacuGPL._SL1050_.jpg", inventory: 100)
+        @bad_paper = @meg.items.create(name: "Bad Paper", description: "Bad for writing on!", price: 20, image: "https://cdn.vertex42.com/WordTemplates/images/printable-lined-paper-wide-ruled.png", inventory: 3)
+        @bad_pencil = @meg.items.create(name: "Bad Pencil", description: "You cannot write on paper with it!", price: 2, image: "https://images-na.ssl-images-amazon.com/images/I/31BlVr01izL._SX425_.jpg", inventory: 100)
+        @bad_candy = @meg.items.create(name: "Bad candy", description: "You cannot eat it!", price: 3, image: "https://images-na.ssl-images-amazon.com/images/I/610ZOacuGPL._SL1050_.jpg", inventory: 100)
+        @very_bad_paper = @meg.items.create(name: "Very Bad Paper", description: "Horrible for writing on!", price: 20, image: "https://cdn.vertex42.com/WordTemplates/images/printable-lined-paper-wide-ruled.png", inventory: 3)
```

Comment:
I need to know about this very bad paper ;)

Reply:
Lol it's super bad